LongWarOfTheChosen/Src/LW_AlienPack_Integrated/Classes/X2Ability_LWAlienAbilities.uc

```diff
@@ -180,9 +180,8 @@ static function X2AbilityTemplate CreateMutonM2_LWAbility_BayonetCharge()
 	ActionPointCost.bConsumeAllPoints = true;
 	Template.AbilityCosts.AddItem(ActionPointCost);
 
-	Cooldown = new class'X2AbilityCooldown';
-	Cooldown.iNumTurns = default.BAYONETCHARGE_COOLDOWN;
-	Template.AbilityCooldown = Cooldown;
+	
+
```

Comment:
Do we need all these blank lines? I know I'm being anal about this, but tidy code is easier-to-read-and-comprehend code.